lib/src/storage/backend/rocksdb.rs

```diff
@@ -464,13 +602,30 @@ impl Db {
         key: &[u8],
     ) -> Result<Option<PinnableSlice>, StorageError> {
         unsafe {
-            let slice = ffi_result!(rocksdb_transactiondb_get_pinned_cf_with_status(
-                self.0.db,
-                self.0.read_options,
-                column_family.0,
-                key.as_ptr() as *const c_char,
-                key.len()
-            ))?;
+            let slice = match &self.0.db {
+                InnerDbHandler::Secondary(inner_db_handler) => {
+                    let mut errptr: Vec<i8> = vec![];
```

Comment:
We never read it to check if an error happend or not.

Reply:
missed that one. created another `with_status` function.